tests/multipleBackend/routes/routeBackbeat.js

```diff
@@ -2540,12 +2586,13 @@ describe('backbeat routes', () => {
                 }, err => {
                     // should error out as location shall no longer exist
                     assert(err);
+                    assert.strictEqual(err.statusCode, 503);
                     done();
                 }),
             ], done);
         });
 
-        itSkipCeph('should batch delete a versioned AWS location', done => {
+        itSkipCephS3C('should batch delete a versioned AWS location', done => {
```

Comment:
condition should be on S3C, but on `AWS` availability

Reply:
Fixed: [7ee63f5](https://github.com/scality/cloudserver/pull/5907/commits/7ee63f5f2fffa27d1cebac8971daf28a6a4ab5b7)